src/clj/game/cards/identities.clj

```diff
@@ -1051,12 +1051,14 @@
 
    "Nero Severn: Information Broker"
    {:events [{:event :encounter-ice
-              :req (req (has-subtype? target "Sentry"))
-              :interactive (req true)
-              :once :per-turn
-              :msg "jack out"
-              :async true
-              :effect (effect (jack-out eid))}]}
+              :optional {
```

Comment:
Can you move the `{` to the beginning of the next line and then line everything up so it's just indented 1 space? As seen in the New Angeles Sol implementation below.